### PR TITLE
Add push event documentation

### PR DESCRIPTION
**Pull request by Andre601**
## Checks

I have...

- [x] read and understood the [Contributing Guidelines][contributing]
- [x] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [ ] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [x] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [ ] **Other:** `__________` 

## Description

Documentation for the Push Event changes. This PR targets the `push-event` branch so that we can update the docs alongside the actual workflow when it's ready.

Closes #

[contributing]: https://github.com/Readme-Workflows/.github/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by Andre601** [comment]
Sorry for all the spam, but I'm in the process of fixing the commenter action here....

File: docs/Configuration.md [hunk 1]
```diff
@@ -363,6 +364,17 @@ This text is displayed whenever you close a Pull request without merging it.
 
 This text is displayed whenever you merge a Pull request.
 
+#### `push`
+> **Default:** `⬆️ Pushed {AMOUNT} commit(s) to {REPO}`
+> 
+> **Supported Placeholders:**
+> - `{AMOUNT}`
+> - `{REPO}`
+> - [`{URL}`](#url_text)
```
Comment (abhijoshi2k) [inline docs/Configuration.md:373]:
```suggestion
```
`{URL}` has been removed for this event.

File: docs/Configuration.md [hunk 2]
```diff
@@ -363,6 +364,17 @@ This text is displayed whenever you close a Pull request without merging it.
 
 This text is displayed whenever you merge a Pull request.
 
+#### `push`
```
Comment (abhijoshi2k) [inline docs/Configuration.md:367]: This event is not added in the list present in `Messages` section.

**Review by abhijoshi2k — CHANGES_REQUESTED** [review]
Available placeholders in `Messages` section does not include `{AMOUNT}`

**Comment by abhijoshi2k** [comment]
`push-event` branch is ready.
Please merge this branch after changes and then merge the `push-event` branch.
Release new version after that please.
Thanks.

**Comment by abhijoshi2k** [comment]
> > Available placeholders in `Messages` section does not include `{AMOUNT}`
> 
> I don't get what you mean... You clearly have `{AMOUNT}` in the added event as shown in the "Files changed" tab on your PR...

I'm talking about line 222 (refer file from this branch please)

**Comment by abhijoshi2k** [comment]
I'm unable to suggest changes on that line. IDK why.

You can find the line having content:
Available Placeholders are: `{DATE}`, `{ID}`, `{FORK}`, `{REPO}`, `{URL}` and `{WIKI}`

This is to be changed to :
Available Placeholders are: `{DATE}`, `{ID}`, `{FORK}`, `{REPO}`, `{URL}`, `{AMOUNT}` and `{WIKI}`

**Comment by abhijoshi2k** [comment]
It's a very small change. If you allow me to push to this branch, I'll do it.
Just let me know

**Pull request merged**